modules/admin/con-rbac-config-permission-policies-external-file.adoc

```diff
@@ -36,7 +36,8 @@ You can add your `policy.csv` file to the {product-short} Helm Chart by creating
 * You are logged in to your {ocp-short} account using the {ocp-short} web console.
 * {product} is installed and deployed using Helm Chart.
 +
-For more information about installing the {product} on {ocp-short} using Helm Chart, see xref:proc-install-rhdh-ocp-helm_admin-rhdh[]. 
+//For more information about installing the {product} on {ocp-short} using Helm Chart, see xref:proc-install-rhdh-ocp-helm_{context}[].
```

Comment:
Will cross-references be handled in a separate PR?

Reply:
All impacted cross-links were fixed as part of this PR. Some have been temporarily removed and will be replaced when there is a URL available for the installation guide. NOTE: Impacted cross-links are xrefs that broke during the local build. It would be a good idea to perform a more extensive audit of relevant cross-links.